man/binom_lik.Rd

```diff
@@ -4,12 +4,15 @@
 \alias{binom_lik}
 \title{Likelihood object for Binomial error distribution}
 \usage{
-binom_lik(y, n)
+binom_lik(y, n, link = c("identity", "logit"))
 }
 \arguments{
 \item{y}{Binomial observations}
 
 \item{n}{Binomial number of trials}
+
+\item{link}{Link function. The "identity" link directly puts unimodal prior on Binomial success
```

Comment:
is it also the case that the posterior is on p in the first case and logit p in the second?